items/src/test/scala/weco/api/items/services/ItemUpdateServiceTest.scala

```diff
@@ -150,8 +151,17 @@ class ItemUpdateServiceTest
     status = CatalogueAccessStatus.Open
   )
 
-  val notRequestableAccessCondition = DisplayAccessCondition(
-    method = CatalogueAccessMethod.NotRequestable
+  val availableDates = Some(
+    List(
+      AvailabilitySlot(
+        ZonedDateTime.parse(
+          "2024-02-29T13:32:44.943107Z[Europe/London]"
```

Comment:
I think - but I'm definitely open to having my mind changed - the API should return _either_ a UTC ISO8601 timestamp, or an ISO8601 timestamp with an hour offset. The timezone name in square brackets thing is non-standard and so I feel it's best to avoid exposing it... Maybe start a thread in Slack to share the question with the team?

Reply:
That was a bit of an oversight, I agree with UTC ISO8601 timestamp 👍 
That weird format is java ZonedDateTime